summer-school.md

```diff
@@ -15,18 +15,18 @@ style="width: 100%;"/>]({{ site.baseurl }}/img/summerschool-campus.png)
 **Nengo with Loihi:**
```

Comment:
Do we still want "Nengo with Loihi" to be the main heading here? Or just something more general about Nengo/neuromorphics?